test/MsBuildScript.Tests/runtests.target

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" DefaultTargets="RunMsBuildScriptTests" 
```

Comment:
nit: we shouldn't need toolsversion or xlmns for msbuild 15+, which we can rely on.